### ISSUE TITLE
NTR: Succinyladenosine high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8808]
New term request Succinyladenosine high in urine (segal_030514193708): Succinylpurines; S-Ado  UMLS: 2676170

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000113
- label: Elevated urinary succinyladenosine level
- parents: [Abnormal urinary purine level (HP:0033161)](https://hpo.jax.org/app/browse/term/HP:0033161)
- pmid: PMID:23937257
- def: The amount of succinyladenosine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8808
- synonyms:
